stable/rabbitmq-ha/templates/alerts.yaml

```diff
@@ -3,12 +3,12 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}-rabbitmq-alerts
-  namespace: monitoring
+  namespace: {{ .Values.prometheus.operator.serviceMonitor.namespace }}
   labels:
-    app: "rabbitmq"
-    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
-    heritage: {{ .Release.Service }}
```

Comment:
Won’t label changes break upgrades?

Reply:
Maybe bump major or minor version to indicate it's a breaking change?

Reply:
As long at the change isn't in the matchLabels or other place where the labels are immutable it should be ok.